### PR TITLE
added logging formatting options

### PR DESCRIPTION
**Pull request by shikharsg**
The format of the worker logs cannot be changed as of now. This pull request attempts to change that. See issue [#959](https://github.com/rq/rq/issues/959).
Not sure if these changes are the most elegant solution. Maybe we should allow the user to pass in a log handler to the worker? Much like how you can pass in an exception handler to a worker.

### CODE REVIEW

**Comment by selwin** [comment]
This PR looks good. Could you also document these two options in this file? https://github.com/rq/rq/blob/master/docs/docs/workers.md

File: rq/logutils.py [hunk 1]
```diff
@@ -5,14 +5,16 @@
 import logging
 
 from rq.utils import ColorizingStreamHandler
+from rq.defaults import (DEFAULT_LOGGING_FORMAT,
+                         DEFAULT_LOGGING_DATE_FORMAT)
 
 
-def setup_loghandlers(level):
+def setup_loghandlers(level, datefmt=DEFAULT_LOGGING_DATE_FORMAT,
```
Comment (selwin) [inline rq/logutils.py]: One last minor detail, could you please change the `datefmt` to `date_format` and `logfmt` to `log_format`? This should keep the variable naming consistent across RQ's code base.

Comment (shikharsg) [reply rq/logutils.py]: Sorry my bad. Fixed now

**Comment by selwin** [comment]
Thanks!

**Pull request merged**